src/to-be-disabled.js

```diff
@@ -1,9 +1,9 @@
-import compose from 'ramda/src/compose';
-import defaultTo from 'ramda/src/defaultTo';
-import includes from 'ramda/src/includes';
-import path from 'ramda/src/path';
-import propEq from 'ramda/src/propEq';
-import anyPass from 'ramda/src/anyPass';
+import compose from 'ramda/src/compose.js';
```

Comment:
You should normally be able to import these files without file extension

```suggestion
import compose from 'ramda/src/compose';
```

Reply:
Without `.js` the tests under Node 14+ are failing with:
```
ENOENT: no such file or directory, open '/Users/maciej/Dev/jest-native/node_modules/ramda/src/compose'
```

Reply:
Due to the same error occurring in apps using Jest Native (see #82), I already had to release 4.0.8 with fixed import paths.